src/command_modules/azure-cli-feedback/azure/cli/command_modules/feedback/custom.py

```diff
@@ -41,10 +41,10 @@
 _QUESTIONS_URL = "aka.ms/azcli/questions"
 
 _CLI_ISSUES_URL = "aka.ms/azcli/issues"
-_RAW_CLI_ISSUES_URL = "https://github.com/Azure/azure-cli/issues/new"
+_RAW_CLI_ISSUES_URL = "https://github.com/azure/azure-cli/issues/new"
```

Comment:
Kept it this way to avoid converting to lower all the time for comparison. We only use this for browser open and not printing so it should be ok I guess.